lib/manager.js

```diff
@@ -86,16 +86,40 @@ class Manager extends EventEmitter {
     this.wsPool = []
     this.authArgs = { dms, calc }
 
+    this._createRestClient()
+  }
+
+  _createRestClient () {
     this.rest = new RESTv2({
```

Comment:
hm, is the rest client used at all in `bfx-api-node-core`? else we might even be able to remove it...

could be also a separate PR in case we can remove it...

Reply:
I'll take a look

Reply:
not used by `api-node-core` nor `hf-algo`, I've removed it